dragonfly/subdivide.py

```diff
@@ -0,0 +1,105 @@
+"""Utilities to interpret parameters for subdividing biulding masses."""
+
+
+def interpret_floor_height_subdivide(floor_to_floor_heights, max_height,
```

Comment:
Both functions in this module can be written as classes. `HeightSubDParam` and `CorePerimeterSubDParam`.

Reply:
This is an interesting idea. I have opened an issue about it: https://github.com/ladybug-tools/dragonfly-core/issues/11